tests/cases/standalone/common/create/create_metric_table.result

```diff
@@ -48,6 +48,11 @@ CREATE TABLE t5 (ts timestamp time index, valval double, host string primary key
 
 Error: 1004(InvalidArguments), Adding field column valval to physical table
 
+-- create logical table with different time unit on time index column
+CREATE TABLE t6 (ts timestamp(6) time index, job string primary key, val double) engine = metric with ("on_physical_table" = "phy");
```

Comment:
Not particularly related to this PR but why don't we create the time index column automatically for logical table? Then we don't even bother checking the time unit.

Reply:
We usually create this table automatically. I just found this issue occasionally. It may not be a big problem.

Currently, logical tables mostly behave like a normal table, we don't expect users to create logical tables manually.